src/views/MidiView.vue

```diff
@@ -178,9 +176,54 @@ onUnmounted(() => {
               />
             </span>
           </div>
-          <div class="control checkbox-container">
-            <input type="checkbox" id="midi-velocity" v-model="midi.velocityOn" />
-            <label for="midi-velocity">Use velocity</label>
+          <div class="control checkbox-group">
+            <span>
+              <input type="checkbox" id="midi-velocity" v-model="midi.velocityOn" />
+              <label for="midi-velocity"> Use velocity </label>
+            </span>
+            <span>
+              <input type="checkbox" id="multichannel" v-model="midi.multichannelToEquave" />
+              <label for="multichannel"> Multichannel-to-equave </label>
+            </span>
+          </div>
+          <label>Settings for multichannel-to-equave mode</label>
+          <div class="control multichannel-input">
+            <span>
```

Comment:
Semantically spans are for *phrasing* content. The styling on the input element puts it on a separate line so in terms of code style this should be a div.